CodingTracker.AnaClos/CodingTracker.AnaClos/Validation.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Globalization;
+
+namespace AnaClos.CodingTracker;
+
+public class Validation
+{
+    public bool Validate2Dates(string startTime,string endTime)
+    {
+        bool ok;
+        var codingSession = new CodingSession { StartTime = startTime, EndTime = endTime };
+        codingSession.CalculateDuration();
+        if (codingSession.Duration.TotalSeconds > 0)
+        {
+            ok = true;
+        }
+        else
+        {
+            ok = false;
+        }
+        return ok;
+    }
+
+    public bool ValidateDate(string date)
+    {
+        bool ok = false;
+        DateTime dateTime = DateTime.MinValue;
+
+        if (date == "r")
+        {
+            ok = true;
+        }
+        else
```

Comment:
Here it would be better to use the TryParse method, which handles unsuccessful scenarios without the need for exceptions. In general you don't want to use Exceptions for control flow (they'e expensive).
![image](https://github.com/user-attachments/assets/b011e467-130d-4c15-9385-0a2d8205d3d2)

Reply:
Thanks for helping me improve my code.